apps/server/src/modules/files-storage/controller/api-test/files-storage-copy-files.api.spec.ts

```diff
@@ -77,37 +77,21 @@ class API {
 	}
 }
 
-const createRndInt = (max) => Math.floor(Math.random() * max);
-
 describe(`${baseRouteName} (api)`, () => {
 	let app: INestApplication;
 	let em: EntityManager;
 	let currentUser: ICurrentUser;
 	let api: API;
-	let s3instance: S3rver;
 
 	beforeAll(async () => {
-		const port = 10000 + createRndInt(10000);
-		const overridetS3Config = Object.assign(s3Config, { endpoint: `http://localhost:${port}` });
-
-		s3instance = new S3rver({
-			directory: `/tmp/s3rver_test_directory${port}`,
-			port,
-		});
-
-		await s3instance.run();
 		const module: TestingModule = await Test.createTestingModule({
 			imports: [FilesStorageTestModule],
-			providers: [
-				FilesStorageTestModule,
-				{
-					provide: 'S3_Config',
-					useValue: overridetS3Config,
-				},
-			],
+			providers: [FilesStorageTestModule],
```

Comment:
It shouldn't be necessary to have this as a provider because it is an import. What does it even mean to have a module as a provider? I don't think that makes any sense any way. A provider is a class that can be injected, but you don't inject a module.